app/client/components/dashboard/metricsBar.jsx

```diff
@@ -21,41 +21,43 @@ const mapDispatchToProps = (dispatch) => {
 };
 
 const MetricsBar = ({creditTransferStats}) => {
-
-    let beneficiaryTermPlural = window.BENEFICIARY_TERM_PLURAL;
-
-    if (Object.keys(creditTransferStats).length == 0) {
-      return (
-        <p>No Transfer Data</p>
-      );
-    }
-
+  if (Object.keys(creditTransferStats).length == 0) {
```

Comment:
`===` javascript is the worrrsttt

Reply:
I've started writing `!==` everywhere in my python code and wondering why it doesn't work.